dask_searchcv/model_selection.py

```diff
@@ -106,12 +108,13 @@ def build_graph(estimator, cv, scorer, candidate_params, X, y=None,
     candidate_params_name = 'cv-parameters-' + main_token
     dsk[candidate_params_name] = (decompress_params, fields, params)
     dsk[cv_results] = (create_cv_results, scores, candidate_params_name,
-                       n_splits, error_score, weights)
+                       n_splits, error_score, weights, multimetric)
```

Comment:
Following my comment on `multimetric` in the `fit` call, this would then `metrics` instead of `multimetric`, as follows:

```
if multimetric:
    metrics = list(scorer.keys())
else:
    metrics = None
```